ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-master.yaml

```diff
@@ -66,6 +66,10 @@ tag_specification:
   name: "4.8"
   namespace: ocp
 tests:
+- as: e2e-openstack
+  steps:
+    cluster_profile: openstack-vexxhost
+    workflow: openshift-e2e-openstack-minimal
```

Comment:
If you want to run the full conformance (parallel) suite, you can copy-paste this:

https://github.com/openshift/release/blob/9c49e8e6afb3ef7f219dea8e38c6a200a7e00896/ci-operator/config/openshift/installer/openshift-installer-master.yaml#L311-L314

Plus, you get "e2e-openstack-ipi" as a name, so that it's consistent with the other instances of it.